.cirrus.yml

```diff
@@ -389,7 +389,6 @@ asan_sanitizer_task:
   env:
     CXXFLAGS: -DZEEK_DICT_DEBUG
     ZEEK_CI_CONFIGURE_FLAGS: *ASAN_SANITIZER_CONFIG
-    ZEEK_CI_DISABLE_SCRIPT_PROFILING: 1
```

Comment:
The reason this was disabled on the asan and ubsan builds is because the profiler code can make the tests no those builds take a very long time. Is that not the case anymore?

Reply:
I believe this topic came up a few months ago and there was barely an effect. I'm usually running btests with ASAN here, so :man_shrugging:

Reply:
Must have been something with an older distro version we were using at the time, I guess. It's been a couple of years since we added it: https://github.com/zeek/zeek/pull/1787

Reply:
A comment from that PR:

> Currently on master, the sanitizer build takes just over an hour. During testing of this change, it was taking just under 40 minutes.

Things must have changed quite a bit since then :-)

![image](https://github.com/zeek/zeek/assets/2100509/a0325990-5091-4a0d-9c3d-b685836b03fc)